main.js

```diff
@@ -16,37 +16,53 @@
   var updateButton = document.querySelector('#update-button');
   var inputForRange = document.querySelector('#set-range-box');
   var randomNumber = createNumber();
+  var winningSide = document.querySelector('#winning-side');
 
   function createNumber() {
     return Math.round(Math.random() * 100);
   }
-  console.log(randomNumber)
+
+  function disableButtons() {
```

Comment:
created functions to plug into other functions further down in the code to reduce line lengths